tests/test_agreements_pool.py

```diff
@@ -99,3 +104,37 @@ def use_agreement_cb(_agreement):
 
     result = await pool.use_agreement(use_agreement_cb)
     assert result is None
+
+
+@pytest.mark.skipif(sys.version_info < (3, 8), reason="AsyncMock requires python 3.8+")
+@pytest.mark.asyncio
+@pytest.mark.parametrize(
+    "multi_activity,simulate_race,event_emitted",
+    [
+        (True, False, True),
```

Comment:
What are these values used for?

Reply:
@VanDavv these are the parameters for a given test run https://docs.pytest.org/en/6.2.x/parametrize.html